src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/HiddenFieldCheck.java

```diff
@@ -344,9 +333,8 @@ private boolean isIgnoredParam(DetailAST ast, String name) {
      * @param name identifier of token
      * @return true if static or instance field
      */
-    private boolean isStaticOrInstanceField(DetailAST ast, String name) {
-        return frame.containsStaticField(name)
-                || !isInStatic(ast) && frame.containsInstanceField(name);
+    private boolean isInstanceField(DetailAST ast, String name) {
```

Comment:
Javadoc still says `static or instance` but code is only dealing with instances.

Reply:
fixed